prompting/tools/datasets/math.py

```diff
@@ -90,12 +91,12 @@ def get(
                 }
 
     def search(
-        self, name, selector: Selector, include: List = None, exclude: List = None
+        self, name : str, selector: Selector, include: Optional[List] = None, exclude: Optional[List] = None
     ) -> Dict:
         raise NotImplementedError(
             f"Search is not implemented for {self.__class__.__name__}"
         )
 
-    def random(self, selector: Selector, **kwargs):
+    def random(self, selector: Selector, **kwargs)->Dict:
```

Comment:
What are the types for keys and values?
```suggestion
    def random(self, selector: Selector, **kwargs) -> dict[str, Any]:
```